assets/panel/debugger.properties

```diff
@@ -488,26 +488,22 @@ editor.enableBreakpoint=Enable breakpoint
 # for removing a breakpoint on a line.
 editor.removeBreakpoint=Remove breakpoint
 
-# LOCALIZATION NOTE (editor.editBreakpoint): Editor gutter context menu item
-# for setting a breakpoint condition on a line.
-editor.editBreakpoint=Edit breakpoint
-
-# LOCALIZATION NOTE (editor.addConditionalBreakpoint): Editor gutter context
+# LOCALIZATION NOTE (editor.addConditionBreakpoint): Editor gutter context
 # menu item for adding a breakpoint condition on a line.
-editor.addConditionalBreakpoint=Add condition
-editor.addConditionalBreakpoint.accesskey=c
+editor.addConditionBreakpoint=Add condition
+editor.addConditionBreakpoint.accesskey=c
```

Comment:
Open question from the other PR: is the accesskey shared across different labels?

Reply:
No it is not at the moment.

Reply:
Thanks, I couldn't figure it out from the code. In case, sharing accesskeys across different labels is a no-go ;-)